### PR TITLE
Add IMO Token

### PR DESCRIPTION
**Pull request by eddy-1**

### CODE REVIEW

**Comment by catenocrypt** [comment]
As per our updated policy, brand new tokens are not included, there is a minimum requirement of 100 holder addresses and 500 transactions, in order to filter out scam tokens.
See https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863#how-to-get-approved
Token check:  Usage is low, partially matches (36 addresses, 10474 transfers).   Name matches.
Putting on hold for now.

**Comment by vikmeup** [comment]
As per our updated policy, brand new tokens are not included, there is a minimum requirement 

- at least 100 token holders
- at least 500 transactions

See our [How to Get Submit Token Logo ](https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863#how-to-get-approved) guidelines.

Feel free to re-open once you meet those requirements.

**Comment by eddy-1** [comment]
100+ token holders
10000+ transactions
Please reopen this pull request.
Thank you very much! @vikmeup

**Comment by eddy-1** [comment]
@vikmeup Meet those requirements. Please reopen this pull request.

**Comment by vikmeup** [comment]
Can you also add `info.json` file? Example: https://github.com/trustwallet/assets/blob/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/info.json

**Comment by eddy-1** [comment]
Yes I have added info.json @vikmeup

File: blockchains/ethereum/assets/0xbAe226690D7DCe1Da1823f0BF811529b8229134a/info.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "IMO",
+    "website": "https://www.imoex.top",
+    "short_description": "IMO Token is the platform token of IMO ecosystem, which enables the project to obtain investment in stages through continuous limited private sales. The project will continue to be privately funded with development progress and stage results. The general private sale periods are divided into 3 months, 6 months, 1 year or even longer. IMO uses a variety of advanced mathematical models to control the private sales and market at different stages. During the private sale period, the exchange and investors will observe the development of the project and decide whether to invest or not. Invest or withdraw. Phased batch-to-batch investments will greatly reduce investor’s risk. At the same time, it also gives encouragement and benign supervision to the project team, so that the project can develop healthily and ultimately return to investors. The IMO platform will strictly screen and check the quality of each project and take certain measures to ensure the interests of investors.",
```
Comment (vikmeup) [inline blockchains/ethereum/assets/0xbAe226690D7DCe1Da1823f0BF811529b8229134a/info.json:4]: Can you make it shorter?

File: blockchains/ethereum/assets/0xbAe226690D7DCe1Da1823f0BF811529b8229134a/info.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "IMO",
+    "website": "https://www.imoex.top",
```
Comment (vikmeup) [inline blockchains/ethereum/assets/0xbAe226690D7DCe1Da1823f0BF811529b8229134a/info.json:3]: can you present information from `imoex.top` and link to official token address?

Comment (eddy-1) [reply blockchains/ethereum/assets/0xbAe226690D7DCe1Da1823f0BF811529b8229134a/info.json:3]: I have present pull request and token information on **imoex.top** . 
[https://www.imoex.top/en_US/noticeInfo/10280](https://www.imoex.top/en_US/noticeInfo/10280)

File: blockchains/ethereum/assets/0xbAe226690D7DCe1Da1823f0BF811529b8229134a/info.json (hunk 1 above)
Comment (eddy-1) [reply blockchains/ethereum/assets/0xbAe226690D7DCe1Da1823f0BF811529b8229134a/info.json]: OK made it shorter.

**Pull request merged**